.github/workflows/claude.yml

```
@@ -40,9 +40,751 @@ jobs:
           additional_permissions: |
             actions: read
 
-          # Optional: Give a custom prompt to Claude. If this is not specified, Claude will perform the instructions specified in the comment that tagged it.
-          # prompt: 'Update the pull request description to include a summary of changes.'
+          # Optional: Give a custom prompt to Claude. If 
+          <!doctype html>
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)